lua/entities/gmod_wire_expression2/core/player.lua

```diff
@@ -577,16 +577,19 @@ end)
 
 
```

Comment:
its a line.... That is there.... Did you expect it to have some magical meaning to life? Or the answer to life itself?